templates/template-kubernetes-sre-copilot/README.md

```
@@ -0,0 +1,206 @@
+  required, see [Getting started](#getting-started))
+
+## Getting started
+
+```shell
+npm install
+cp .env.example .env
+```
+
+Fill in your `.env`:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)